Formula/cod.rb

```diff
@@ -0,0 +1,20 @@
+class Cod < Formula
+  desc "Completion daemon for bash/fish/zsh"
+  homepage "https://github.com/dim-an/cod"
+  url "https://github.com/dim-an/cod/archive/refs/tags/v0.1.0.tar.gz"
+  sha256 "3d8ed6f284afcf4c86a2164e234ab7ff40c50aa6ab0bd892e59f8dc8aef02541"
+  license "Apache-2.0"
+
+  depends_on "go" => :build
+
+  def install
+    system "make", "build"
+    bin.install "cod"
+  end
+
+  test do
+    assert_match "", shell_output("cod daemon")
+    assert_match "", shell_output("cod remove cod")
```

Comment:
Matching nothing seems kinda useless

Reply:
My assumption here is `shell_output` will throw an exception / fail if a non-zero exit code is encountered. Is that true?

I can change these to `asset(shell_output(...)).blank?` which is more what I'm looking for here, assuming exit codes are checked.

Reply:
`shell_output()` takes an optional return code to match against: https://rubydoc.brew.sh/Homebrew/Assertions.html#shell_output-instance_method

Reply:
> I can change these to asset(shell_output(...)).blank? which is more what I'm looking for here, assuming exit codes are checked.

Yes they are, please do